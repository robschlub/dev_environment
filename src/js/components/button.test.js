import React from 'react';
import renderer from 'react-test-renderer';
import Button from './button';

describe('Button', () => {
  // test('dummy', () => {
  //   expect(true).toBe(true);
  // });
  test('Should render correctly', () => {
    const component = renderer.create(<Button />);
    const tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });
});

