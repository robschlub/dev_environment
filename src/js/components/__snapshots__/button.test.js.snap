// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Button class is defined, but not used 1`] = `
<button
  className="btn"
>
  hello
</button>
`;

exports[`Button Button class used without btn 1`] = `
<button
  className="btn btn-test-class"
>
  hello
</button>
`;

exports[`Button Children are used to label button 1`] = `
<button
  className="btn"
>
  hello
</button>
`;

exports[`Button Label is used to name button 1`] = `
<button
  className="btn"
>
  hello
</button>
`;

exports[`Button Label is used when label and children defined 1`] = `
<button
  className="btn"
>
  button label
</button>
`;

exports[`Button No props 1`] = `
<button
  className="btn"
>
  Button
</button>
`;

exports[`Button href is used 1`] = `
<a
  className="btn"
  href="/#"
>
  hello
</a>
`;
